js/ui/widget/ui.widget.js

```diff
@@ -491,7 +489,7 @@ var Widget = DOMComponentWithTemplate.inherit({
     _attachHoverEvents() {
         const { hoverStateEnabled } = this.option();
         const selector = this._activeStateUnit;
-        const namespace = UI_FEEDBACK;
+        const namespace = 'UIFeedback';
```

Comment:
```suggestion
```

I think it's better to use one constant
```js
hover.off($el, { selector, namespace: UI_FEEDBACK });
<...>
active.off($el, { selector, namespace: UI_FEEDBACK });
```

Reply:
Why? :) Can I use different namespaces for the different events?

Reply:
You use the same namespace twice in this file. Using a constant prevents the possibility of being sealed when using this namespace again

Reply:
But it is not a constant :) Why these different namespaces should have general value?

Reply:
> Why these different namespaces should have general value

Why are they different? They have the same value
[L492](https://github.com/DevExpress/DevExtreme/pull/10747/files#diff-de09fda2ed9e5fc71c891890679f5cc1R492): `const namespace = 'UIFeedback';`
[L513](https://github.com/DevExpress/DevExtreme/pull/10747/files#diff-de09fda2ed9e5fc71c891890679f5cc1R513): `const namespace = 'UIFeedback';`